compiler-rt/lib/asan/asan_poisoning.h

```diff
@@ -10,15 +10,64 @@
 //
 // Shadow memory poisoning by ASan RTL and by user application.
 //===----------------------------------------------------------------------===//
+#ifndef ASAN_POISONING_H
+#define ASAN_POISONING_H
 
 #include "asan_interceptors.h"
 #include "asan_internal.h"
 #include "asan_mapping.h"
 #include "sanitizer_common/sanitizer_flags.h"
 #include "sanitizer_common/sanitizer_platform.h"
+#include "sanitizer_common/sanitizer_ring_buffer.h"
+
+// For platforms which support slow unwinder only, we restrict the store context
+// size to 1, basically only storing the current pc. We do this because the slow
+// unwinder which is based on libunwind is not async signal safe and causes
+// random freezes in forking applications as well as in signal handlers.
+#define GET_STORE_STACK_TRACE_PC_BP(pc, bp)                            \
+  UNINITIALIZED BufferedStackTrace stack;                              \
+  int max_stack = 16;                                                  \
+  if (!SANITIZER_CAN_FAST_UNWIND)                                      \
+    max_stack = Min(max_stack, 1);                                     \
+  stack.Unwind(pc, bp, nullptr, common_flags()->fast_unwind_on_malloc, \
+               max_stack);
+
+#define GET_STORE_STACK_TRACE \
```

Comment:
Why can't we use macros from compiler-rt/lib/asan/asan_stack.h ?
Looks like GET_STACK_TRACE or GET_STACK_TRACE_MALLOC

Reply:
Done - now using GET_STACK_TRACE